index.js

```diff
@@ -125,6 +149,15 @@ class WdioTeamcityReporter extends WdioReporter {
    * @param {SuiteStats} suiteStats
    */
   onSuiteEnd (suiteStats) {
+    const pendingTests = Object.values(suiteStats.tests).filter(test => test.state === 'pending')
+    pendingTests.forEach(testStat => {
+      testStat.error = testStat.error || {
+        message: '',
```

Comment:
This one reports pending tests as failed. How do you think, may be we can put a default message here instead?

Reply:
Good point. I added a default message.